extensions/agroal/runtime/pom.xml

```diff
@@ -90,6 +90,19 @@
                         <provides>io.quarkus.agroal</provides>
                     </capabilities>
                 </configuration>
+                <executions>
+                    <execution>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>extension-descriptor</goal> 
+                        </goals>
+                        <configuration>
+                            <conditionalDevDependencies> 
+                                <artifact>${project.groupId}:${project.artifactId}-dev:${project.version}</artifact> 
```

Comment:
While it's cool you can do that, the ` <artifactId>quarkus-agroal-dev</artifactId>` is hardcoded anyway and if someone is looking for usage (or simply trying to figure how it works) of the `quarkus-agroal-dev`, they won't find it. If the `artifactId` ever changes, then also will have to change the `quarkus-agroal-dev` so I think this is unnecessary and it would be easier to read if the value was hardcoded.